### PR TITLE
fix(#patch); pricelib; fix UniswapForksRouter.getLpTokenLiquidityUsdc decimals; add DPR override

### PR DESCRIPTION
**Pull request by dhruv-chauhan**
- Mis-pricing of depler (DPR) token due to low liquidity: https://dexscreener.com/ethereum/0xa14d7186a935a1f07ca247fa9363a8e45e8b68e9
- Fix: 
  - there was a decimal issue in uniswap router's liquidity calculation, fixed for all usage
  - added oracle order override for delper
  ```
  [getLiquidityBoundPrice] token: 0x077416cc6242b3a7d8e42652b8a6a2599fda4a92 (reported price * amount): (43.166591 * 4999999973.145014051472408216) bound to available liquidity: 965.30624801568125952580534; new price: 0.0000001930612506400676865339973194571663
  ```
  - Updated the function `getInputTokenAmountPrice` to separately set a single token's price and its amount multiple
- Deployments:
  - arbitrum-one-bridge-ethereum: https://okgraph.xyz/?q=dhruv-chauhan%2Farbitrum-one-bridge-ethereum
- Other affected subgraphs:
  - arbitrum-one-bridge-arbitrum: https://okgraph.xyz/?q=dhruv-chauhan%2Farbitrum-one-bridge-arbitrum
  - lido-ethereum: https://okgraph.xyz/?q=dhruv-chauhan%2Flido-ethereum
  - optimism-bridge-v2-ethereum: https://okgraph.xyz/?q=dhruv-chauhan%2Foptimism-bridge-v2-ethereum
  - optimism-bridge-v2-optimism: https://okgraph.xyz/?q=dhruv-chauhan%2Foptimism-bridge-v2-optimism

### CODE REVIEW

File: subgraphs/_reference_/src/prices/config/mainnet.ts [hunk 1]
```diff
@@ -152,6 +152,23 @@ class baxaOverride implements OracleConfig {
   }
 }
 
+// https://github.com/messari/subgraphs/issues/2329
+class delperOverride implements OracleConfig {
```
Comment (dmelotik) [inline subgraphs/_reference_/src/prices/config/mainnet.ts:156]: class should be UpperCase https://google.github.io/styleguide/tsguide.html#identifiers

**Review by dmelotik — APPROVED** [review]
small naming fix, but LGTM! Nice work @dhruv-chauhan

**Comment by dmelotik** [comment]
@dhruv-chauhan the deployment you linked (https://okgraph.xyz/?q=dhruv-chauhan%2Farbitrum-one-bridge-ethereum-2) looks like it has the same issue

**Comment by dhruv-chauhan** [comment]
TVL is calculated as, 
`tvl = getInputTokenAmountPrice(inputTokenBalance)`

- Inside `getInputTokenAmountPrice`, we would fetch the price of one token and multiply it by the amountof tokens
- Fetching the price of a single token would bypass our liquidity bounding check, since multiplication by amount is done outside the pricelib
- Updated the function to separately set one token's price and amount's price 

@dmelotik

**Pull request merged**